kubernetes/updater/README.md

```diff
@@ -0,0 +1,23 @@
+## Teleport Kubernetes Updater (`teleport-kube-updater`)
+
+The Teleport kubernetes updater is a controller in charge of updating Teleport Kubernetes agents.
+This alleviates the cost of updating all agents on large-scale deployments.
+
+### Design
```

Comment:
Yup, this will either be part of the `teleport-kube-agent` chart, or a new `teleport-kube-agent-updater` chart that will be an optional dependency fo `teleport-kube-agent`. In the MVP this should be enabled automatically for cloud users only.

Technically you could install this on your own and point it to any custom teleport Deployment or StatefulSet you built, it should update them the same way. (In fact it's almost a generic deployment updater, it's just that version.Getter and maintenance.Triggers are kinda Teleport-specific and not currently configurable).